templates/taxi/car_list.html

```diff
@@ -1,22 +1,26 @@
-{% extends "base.html" %}
-
-{% block content %}
-  <h1>
-    Car list
-    <a href="{% url 'taxi:car-create' %}" class="btn btn-primary link-to-page">
-      Create
-    </a>
-  </h1>
-  {% if car_list %}
-    <ul>
-      {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
-        </li>
-      {% endfor %}
-    </ul>
-  {% else %}
-    <p>There are no cars in taxi</p>
-  {% endif %}
-{% endblock %}
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>
+    Car list
+    <a href="{% url 'taxi:car-create' %}" class="btn btn-primary link-to-page">
+      Create
+    </a>
+  </h1>
+  {% if car_list %}
+    <form method="get", action="{% url 'taxi:search-cars' %}">
+      <input type="text", name="q", placeholder="Search cars (model)">

```

Comment:
There are commas after the `method` and `name` attributes in the form and input tags. These should be removed to ensure proper HTML syntax.